packages/core/src/builder.class.test.ts

```diff
@@ -0,0 +1,15 @@
+import { Builder } from './builder.class';
+
+describe('Builder', () => {
+  test('trustedHosts', () => {
+    expect(Builder.isTrustedHost('example.com')).toBe(false);
+    expect(Builder.isTrustedHost('localhost')).toBe(true);
+    expect(Builder.isTrustedHost('builder.io')).toBe(true);
+    expect(Builder.isTrustedHost('123-review-build.beta.builder.io')).toBe(true);
```

Comment:
can we add a test for the problematic `builder.io` subdomains? `cdn.builder.io`, etc.

Reply:
What about white label instances? Such as `umg-mira.builder.io` or  `marketer.mira.umusic.com` etc. I suppose that is why we have the `registerTrustedHost` method? 

Can we add tests to that to make sure it works as expected? My worry is that if we release this as a patch update it might break some weird edge cases.

Reply:
One more we probably want to add is `playground.builder.io`

Reply:
And `qa.builder.io`, also will this update break support for `www.builder.io`?

Reply:
Should be a minor release, since it potentially involves changes on the user's end to add their hosts